spec/monetize_spec.rb

```diff
@@ -118,6 +118,11 @@
           expect(Monetize.parse('L9.99')).to eq Money.new(999, 'USD')
         end
 
+        it 'ignores ZAR symbols that is part of a text' do
+          expect(Monetize.parse('EUR 9.99')).to eq Money.new(999, 'EUR')
+          expect(Monetize.parse('9.99 EUR')).to eq Money.new(999, 'EUR')
+        end
```

Comment:
This is probably enough to test for that regression.

I wonder if having tests for all ISO codes the same way there's tests for symbols from `Monetize::Parser::CURRENCY_SYMBOLS` would help making this a bit more future-proof.

Reply:
Good point! I thought about this, but it's the "R" symbol that stands out of the crowd here (others are special symbols).

I'm also working on a new parser for this gem that should make it much more predictable and strict (currently there are all sorts of untested edge cases, e.g. `12:35:59`), so this should not be a problem in the future.

Reply:
Yeah, true. The new parser seems promising, good luck with that.

Reply:
Thank you, I'll push a WIP branch shortly